templates/templates/search.html

```diff
@@ -1,5 +1,5 @@
 <form action="/search" id="google-appliance-search-form" class="header-search">
 		<label for="edit-keys" class="accessibility-aid">Search</label>
-    <input type="search" maxlength="255" name="q" id="edit-keys" class="form-text" placeholder="Search" value="{{ query }}" />
+    <input type="search" maxlength="255" name="q" id="edit-keys" class="form-text" placeholder="Search on ubuntu.com" value="{{ query }}" />
```

Comment:
This still says "Search on ubuntu.com"

```suggestion
    <input type="search" maxlength="255" name="q" id="edit-keys" class="form-text" placeholder="Search our sites" value="{{ query }}" />
```